develop/parachain-devs/runtime-development/FRAME/create-custom-pallet.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Create a Custom Pallet
+description: TODO
+---
+
+# Create a Custom Pallet
+
+## Introduction
```

Comment:
Should we also have a prerequisite section?

Reply:
I've added to the introduction this sentence:
`To get the most out of this guide, ensure you're familiar with [FRAME concepts](TODO:update-path){target=\_blank}.`

Do you think that's enough? At this stage of the docs, I don't think adding a prereq like 'Make sure you have rust installed` is relevant.

Reply:
I made a comment about this in my PR review, but it was more just that I don't think that should be the first sentence of the page

But are there any other prerequisites besides knowledge of FRAME concepts?